src/hooks/useV2Pairs.ts

```diff
@@ -48,6 +48,7 @@ export function useV2Pairs(currencies: [Currency | undefined, Currency | undefin
       if (!tokenA || !tokenB || tokenA.equals(tokenB)) return [PairState.INVALID, null]
       if (!reserves) return [PairState.NOT_EXISTS, null]
       const { reserve0, reserve1 } = reserves
+      if (reserve0.isZero() && reserve1.isZero()) return [PairState.NOT_EXISTS, null]
```

Comment:
i wonder if this breaks adding liquidity to a v2 pair that exists but is not initialized

Reply:
yeah i was looking as well, it shouldn't because `addLiquidity` is isomorphic. however we should go through all the cases where `PairState.NOT_EXISTS` is used to drive logic to make sure this is ok. in particular i already found a bug in `AddLiquidityV2 > index.tsx`, the `isCreate` flag is no longer valid

Reply:
Should I just handle checking reserve0/1 = 0 in the caller to avoid changing the behavior everywhere?